jacow/ISO-4 formatter.txsMacro

```diff
@@ -0,0 +1,24 @@
+{
+    "abbrev": "",
+    "description": [
+        "Format the selected text in ISO-4 form."
```

Comment:
This description should be a bit more extended, especially since it calls an external tool/webpage

Reply:
txs users will only see this description ...